app/knight.py

```diff
@@ -0,0 +1,40 @@
+
+from typing import List
+from .armor import Armor
+from .weapon import Weapon
+from .potion import Potion
+
+
+class Knight:
+    def __init__(
+            self,
+            name: str,
+            power: int,
+            hp: int,
+            armour: List[Armor],
+            weapon: Weapon,
+            potion: Potion = None
+    ) -> None:
+        self.name = name
+        self.power = power
+        self.hp = hp
+        self.armour = armour
+        self.weapon = weapon
+        self.potion = potion

```

Comment:
Apply all attributes in init
Also make functions apply_armour, apply_weapon, apply_potion private or protected